recipe/meta.yaml

```diff
@@ -1,73 +1,89 @@
 {% set name = "prody" %}
-{% set version = "2.5.0" %}
+{% set version = "2.5.1" %}
+{% set python_min = "3.10" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  git_url: https://github.com/jamesmkrieger/{{ name }}.git
-  git_rev: 7d21dc4
+  path: /home/vbadwaj/Desktop/ProDy-2.5.0rc1
+  # include:
+  #   - prody/**
+  #   - setup.py
+  #   - pyproject.toml
+  #   - README.*
+  #   - LICENSE.rst
+
 
 build:
-  noarch: python
   number: 4
   entry_points:
     - prody = prody.apps:prody_main
     - evol = prody.apps:evol_main
+    # - script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+    
 
 requirements:
   build:
     # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
     # Compilers are named 'c', 'cxx', 'fortran', among others
-    - python >={{ python_min }},<3.13
+    - python
     - biopython
     - requests
     - setuptools  # requires pkg_resources at runtime
     - scipy
-    - pyparsing <=3.1.1
+    - pyparsing >=3.1.2
     - conda-build
     - pip
-    - numpy >=1.10,<1.25
-    - gfortran
-    - gcc
-    - gxx
+    # - {{ compiler('fortran') }}
+    - numpy >=1.10,<2.0
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ stdlib("c") }}
+    - {{ compiler('fortran') }}
+    - libgfortran5
+    # - gfortran
+    # - gcc
+    # - gxx
+    # - {{ compiler('c') | string }}
+    # - {{ compiler('cxx') | string }}
+    # - {{ "libc"}}
+    # - {{ compiler('fortran') | string }}
+
   host:
-    - python {{ python_min }}
-    - pip <21.1
+    - python 
+    - pip >=22
     - setuptools
-    - numpy >=1.10,<1.25
+    - numpy >=1.10,<2.0
```

Comment:
again, please confirm that you are using a version of prody that can handle this

Reply:
ok, great! 

What is ProDy 2.5.1? Is there a pushed tag and an updated version number inside prody somewhere and updated release notes?